internal/services/delegation.go

```diff
@@ -56,6 +60,41 @@ func (s *Service) processNewBTCDelegationEvent(
 		)
 	}
 
+	stakingTxHash, parseErr := chainhash.NewHashFromStr(delegationDoc.StakingTxHashHex)
+	if parseErr != nil {
+		return types.NewError(
+			http.StatusInternalServerError,
+			types.InternalServiceError,
+			fmt.Errorf("failed to parse staking tx hash: %w", parseErr),
+		)
+	}
+
+	stakingOutputPkScript, parseErr := hex.DecodeString(delegationDoc.StakingOutputPkScript)
+	if parseErr != nil {
+		return types.NewError(
+			http.StatusInternalServerError,
+			types.InternalServiceError,
+			fmt.Errorf("failed to decode staking output pk script: %w", parseErr),
+		)
+	}
+
+	confirmationEvent, registerErr := s.btcNotifier.RegisterConfirmationsNtfn(
```

Comment:
Why do we want to know the confirmation status of the delegation on BTC? At this stage, only eoi is submitted, we don't know whether it will go to BTC. Did I miss anything? cc @jrwbabylonlab

Reply:
I think `2/10 blocks until confirmed` is a nice-to-have feature. For the vanilla indexer, it is better to take it out